src/Exception/Renderer/InlineDebugRenderer.php

```diff
@@ -46,14 +41,19 @@ class InlineDebugRenderer implements RendererInterface
     protected $debug;
 
     /**
-     * @param EngineInterface $templating Templating engine
-     * @param string          $template   Template to render
-     * @param bool            $debug      Whether the debug is enabled or not
-     * @param bool            $forceStyle Whether to force style within the template or not
+     * @var Environment
+     */
+    private $twig;
+
+    /**
+     * @param Environment $twig       Templating engine
+     * @param string      $template   Template to render
+     * @param bool        $debug      Whether the debug is enabled or not
+     * @param bool        $forceStyle Whether to force style within the template or not
      */
-    public function __construct(EngineInterface $templating, $template, $debug, $forceStyle = true)
+    public function __construct(Environment $twig, $template, $debug, $forceStyle = true)
```

Comment:
Can you replace parameters with typehint and remove the comment?

Reply:
But type hint is already exists here. Removing type hint for AbstractBlockService was made for compability with other bundles (for easier upgrade). Should i remove type hint from `Environment $twig` or comment `@param Environment $twig       Templating engine` ?

Reply:
My comment was badly explained, I updated it. So we can typehint all parameters and remove the phpdoc comment.

Reply:
Fixed. It is `public function __construct(Environment $twig, string $template, bool $debug, bool $forceStyle = true)` now without phpdoc comment

Reply:
Great, thank you very much!